lib/zendesk_apps_support/validations/svg.rb

```diff
@@ -22,6 +24,19 @@ def call(package)
           end
           errors
         end
+
+        private
+
+        # to ignore the optional XML declaration at the top of a document
+        def strip_declaration(markup)
+          Nokogiri::XML(markup).root.to_s
+        end
+
+        def are_equivalent_sans_declarations(clean_markup, markup)
```

Comment:
Could the method not just be called `are_equivalent`?  The `sans_declaration` seems like an implementation detail of the method